### DISCUSSION TITLE
Locodock

### DISCUSSION THREAD

**Discussion by ronlobo** [loco-rs/loco#358] [Ideas]
To provide some flexibility for local development, we could have a infrastructure setup similar to [laradock](https://github.com/laradock/laradock) for loco.

Thoughts?

**Comment by kaplanelad** [comment]
@ronlobo, can you elaborate on what is missing in the local development?

**Reply by ronlobo** [reply]
Nothing missing really, laradock just provides a flexible way to add new infrastructure technology to the stack.

**Reply by kaplanelad** [reply]
Feel free to update me if you come across any recommendations within their repository that you suggested we add or modify.

**Reply by jondot** [reply]
@ronlobo we now have Initializers, which allow for a flexible way to add infrastructure to an app.

**Reply by enieber** [reply]
@ronlobo [this docker-compose](https://github.com/loco-rs/loco/blob/master/starters/saas/.devcontainer/docker-compose.yml) help to start create locodock with nginx, grafana, prometheus ands anothers tools.
